lms/djangoapps/organizations/views.py

```diff
@@ -174,9 +175,12 @@ def users(self, request, pk):
                 organization.users.remove(user)
                 users_removed += 1
```

Comment:
@ziafazal if 5 users are passed in and only 1 is in the organization response would be `5 users removed from organization `, while it should be `1 user removed from organization `